### ISSUE TITLE
Using browserslist-stats.json with pnpm and vscode-stylelint

### ISSUE THREAD

**Issue by blvdmitry** [browserslist/browserslist#834]
I'm creating a package that we're planning to use across multiple repositories with a stylelint configuration. A part of it is testing for the browser features support using `browserslist` with GA data extracted to a JSON file. Some of our projects use yarn and there is one using pnpm where this become a bit weird. 

We store our browserslist config in a separate package (`package-name`) and we use the following line. `browserslist-stats.json` is located in the same package and in the same folder but we still have to refer to it through a package name.

```
"last 3 versions and not dead and > 0.5% in package-name stats"
```

When we run stylelint as an npm script – it all works flawlessly, but whenever I'm saving the files in vscode and stylelint plugin is trying to do the same it fails trying to import the stats file. (MODULE_NOT_FOUND)

Trying to log the path in the `node.js` `loadStat` function gives me the correct import path but I've noticed that this error happens only when `require.resolve` there is using `{ paths: ['.'] }` option. Looking through the project history, I've noticed that it's been added there since the moment this feature was implemented so I'm not sure if there was any specific reason behind it:

https://github.com/browserslist/browserslist/commit/b6438bcb61cf6954ef51dc473421654da457df4c

**Comment by ai** [comment]
Without `{ paths: ['.'] }` `require` will look the config from the `node_modules/browserslist`, not from the current working dir.

How do you suggest changing it?

**Comment by blvdmitry** [comment]
Hm, weird, I was expecting it to be the opposite after reading the require.resolve docs:

```
paths [<string[]>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type) Paths to resolve module location from. If present, these paths are used instead of the default resolution paths, with the exception of [GLOBAL_FOLDERS](https://nodejs.org/api/modules.html#loading-from-the-global-folders) like $HOME/.node_modules, which are always included. Each of these paths is used as a starting point for the module resolution algorithm, meaning that the node_modules hierarchy is checked from this location.
```

So not sure if I'm getting it wrong but this gives me an impression that adding `paths: ['.']` means the node_modules resolving starts inside the browserslist config package and doesn't try other paths that can be potentionally used for resolving the module import

**Comment by ai** [comment]
`.` is current working dir, not the `__dirname`

**Comment by ai** [comment]
Check what is `process.cwd()` in CLI and VS Code plugin. Very likely that the problem is with different current working dir.

**Comment by blvdmitry** [comment]
I tried replacing it with process.cwd() too which was not helping. But might also be something specific to my setup of pnpm in combination with vscode-stylelint. So I can just try to resolve it locally on my machine and close the issue for now since I haven't seen anyone else having the same problem and this setup combination doesn't sound like anything new.

**Comment by ai** [comment]
No, I mean do `console.log(process.cwd())` and compare the results
